JudoPayDotNet/Models/CardPaymentModel.cs

```diff
@@ -53,6 +53,15 @@ public class CardPaymentModel : PaymentModel
         /// </value>
         [DataMember(EmitDefaultValue = false)]
         public CardAddressModel CardAddress { get; set; }
+
+        /// <summary>
+        /// Gets or sets the full card holder name.
+        /// </summary>
+        /// <value>
+        /// The card holder name.
+        /// </value>
+        [DataMember(IsRequired = false)]
+        public string CardHolderName { get; set; }
```

Comment:
CardHolderName should probably be applied at the same model levels as MobileNumber and EmailAddress as all will be needed (or not) together.

They shouldn't be needed for  AndriodPaymentModel or PKPaymentModel, but will all be needed for CardPaymentModel, TokenPaymentModel and OneTimePaymentModel